### PR TITLE
656 median mtd

### PR DESCRIPTION
**Pull request by Prerana17**
# Pull Request

Will be review by @0liver0815 initially.

Fixes #https://github.com/Roche/crmPack/issues/656

### CODE REVIEW

File: R/Simulations-methods.R [hunk 1]
```diff
@@ -979,6 +981,15 @@ setMethod("show",
 
 
 
+      cat(
```
Comment (0liver0815) [inline R/Simulations-methods.R:984]: Move the reporting between MostSelected and Stopping.

File: R/Simulations-methods.R [hunk 2]
```diff
@@ -979,6 +981,15 @@ setMethod("show",
...
+        "Median MTD : ",
+        round(unlist(object@additional_stats)[1], 2), "\n"
+      )
+      cat(
+        "CV MTD : ",
```
Comment (0liver0815) [inline R/Simulations-methods.R:989]: Mean CV (MTD)
multiply by 100 and add % as unit.

**Review by 0liver0815 — COMMENTED** [review]
Initial review finished.

**Comment by danielinteractive** [comment]
Hi @Prerana17 is this ready for my review? just let me know and I can do it

**Review by 0liver0815 — COMMENTED** [review]
Just minor changes to comments.

**Comment by 0liver0815** [comment]
To resolve the r cmd check error, I think you need to add an empty list to the other simulation methods, as e.g. done for the stopping here: https://github.com/Roche/crmPack/blob/96702347ab8b7cc09c69df80110afbd1b10bdfd6/R/Design-methods.R#L1093
To avoid empty reporting lines, I propose that the results are only printed in summary, when the list is not empty.

**Comment by Prerana17** [comment]
Old PR for median and CV MTD

**Pull request closed without merging**